CIP-????/README.md

```diff
@@ -0,0 +1,334 @@
+- It does not have a credential
+
+Unresolved inputs and resolving outputs are illegal outside a validation zone. 
+They are included in the script context accurately, with one subtlety: in the same way that normal transaction inputs are decorated with the value that they provide, resolving transaction outputs are similarly decorated with the value that they provide.
+
+### Required transactions
+
+We add a new field to the transaction body, `requiredTxs`, which is a list of transaction IDs.
+
+The `requiredTxs` field is special in that it is _not_ included when hashing the transaction body to create the transaction ID. 
+It is however signed when the transaction body is signed.
+
+The required transactions field is illegal outside a validation zone. 
+It is included in the script context.
+
+### Validating a zone
+
+Validating a zone proceeds as follows:
+
+#### 1: Check all zone-level properties
+The following zone-level properties are checked:
+
+1. (UTXIs are resolved) The unresolved inputs and resolving outputs in the zone match up exactly.
+2. (Linearizability) The transactions are linearizable, that is: there is a reordering of the transactions in the zone such that every resolving output comes before every unresolved input which it resolves.
+3. (Required transactions) For every transaction, all the transactions listed in `requiredTxs` are present in the zone.
+
+If any of these fail, the entire zone is discarded.
+
+#### 2: Validate transactions
+
+All transactions are validated for phase 1 checks; then they are all validated for phase 2.
+
+Resolving outputs are validated similarly to normal transaction inputs: we must check that the credential on the unresolved input accepts the resolving transaction, just as when spending a normal transaction output. 
+
+The other tricky case is where a transaction is phase-2 invalid. 
+To help us with this, we add two additional (phase-1) validity conditions inside the zone:
+
+1. (Safe collateral) All collateral inputs come from outputs outside the zone.
+2. (Excess collateral) The collateral required for a transaction containing scripts is increased: the base amount is now based on the fee for the transaction and _every transaction it depends on_ within the zone.
+    - “A depends on B” here means that A is required in order to determine B’s validity. Note that this means that for unresolved-input-resolving-output pairs the dependency is the opposite way around to the relationship for a normal input-output pair.
+
+If a transaction is phase-2 invalid, then the following occurs:
+
+- The node constructs an invalidity witness for the transaction. This consists of the invalid transaction, along with every transaction that it depends on from within the zone. 
+- The node posts the invalidity witness to the chain and claims the collateral associated with the failing transaction.
+
+## Rationale: how does this CIP achieve its goals?
+ 
+### Design
+
+#### Linearizability and value flow
+
```

Comment:
I know what you are saying here, but if i didnt, something like the following might sound more clear to me : 
All "borrowed" or "backwards-dependent" assets are **backed** by assets that already exist on chain, and are provided by one of the transactions in the zone.